### PR TITLE
mr-46-preparacion-del-proyecto

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -29,6 +30,9 @@ function App() {
             window.removeEventListener('keydown', handleCloseWindow);
         };
     }, [showNavbar]);
+    useEffect(() => {
+        login(12345678, 'asd123123').then((response) => console.log(response));
+    }, []);
```
Comment (nathan-hyan) [inline src/App/App.tsx:35]: This is only for testing purposes, the login function will be used later

Comment (nathan-hyan) [reply src/App/App.tsx:35]: Remove this block of code

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Amazing job! Just one comment

File: src/App/App.tsx (hunk 1 above)
Comment (JaqKent) [reply src/App/App.tsx:35]: done

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**